Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM rocker/r-ver:4.3.1
+
+ENV RENV_CONFIG_REPOS_OVERRIDE "https://cloud.r-project.org/"
+ENV RENV_PATHS_LIBRARY renv/library
+
+# Install system dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    libcurl4-openssl-dev libssl-dev libxml2-dev libgit2-dev git \
+    libudunits2-dev python3-dev python3-pip
+
+# Install pipenv for Python dependencies
+RUN pip install pipenv
+
+# Install renv for R dependencies
+RUN Rscript -e "install.packages('renv')"
```

Comment:
**nitpick:** This is _probably_ redundant since renv will bootstrap itself in the presence of `.Rprofile` files.

**suggestion:** I would change L29 to copy `renv.lock`, `.Rprofile` and `renv/` to get the full bootstrap going.

Reply:
Ah, that makes sense! Done in 3edd026230763494693a033611baae2ec45c87d2 and d1e9ff726c99df0e9129e34d782cec720c36b692.